Client/Extensions/UIWindow+Extension.swift

```diff
@@ -41,3 +41,4 @@ extension UIWindow {
         keyWindow?.windowScene?.statusBarManager?.statusBarFrame.height ?? 0
     }
 }
+// swiftlint:enable first_where
```

Comment:
Yes. New versions of swiftlint throw an error that you disabled, but didn't enable again. So this would have increased the error. Probably won't run into this on BR until we update, but, my machine was yelling at me, so I fixed it. :)